### DISCUSSION TITLE
Question on LineLayer - Need to reorder linelayer on the basis of width

### DISCUSSION THREAD

**Discussion by goelkinshuk** [visgl/deck.gl#6476] [Q&A]
![image](https://user-images.githubusercontent.com/72377458/144723035-01c3ae19-632b-404b-b89f-92c0d6854cac.png)

As you can see transit width is small than car and I want that pink layer to overlap grey wherever it shares the same position.

**Comment by Pessimistress** [comment]
The easiest way to do it is to sort your lines into multiple layers, one for each category.

**Reply by goelkinshuk** [reply]
![image](https://user-images.githubusercontent.com/72377458/144733566-fe50de8c-e1f9-4bb4-bcea-52d8eef13c2f.png)

I am having multiple layers only handling each transport . is there any way I can manage the overlapping somewhere. and sometimes transit width is more also than a car. it has to be dynamic.

**Reply by Pessimistress** [reply]
Layers are drawn in the order that they are defined in the `layers` prop. If you want the wider lines to show on the bottom then you need to sort the layers so that they are drawn first.

**Reply by goelkinshuk** [reply]
Thanks for the clarification. @Pessimistress
